README.md

```diff
@@ -22,6 +22,15 @@ All components below the page level should be props-driven - data fetching shoul
 
 When developing new components, you should create new stories in Storybook for the key states to allow easy review of both visuals and functionality.
 
+## Keycloak setup
+
+- Start database and Keycloak serve by run `docker-compose up`
+- Open [http://localhost:18080/admin](http://localhost:18080/admin/) and log in with the username and password which you can find in the `docker-compose.yml`.
+- Click `master` to open the dropdown menue and click `Create Realm`
```

Comment:
We should be able to skip this step entirely by importing a dev setup to keycloak - see "Importing A Realm On Startup" at https://www.keycloak.org/server/containers